### ISSUE TITLE
Accessibility Testing for [MHV, Secure Messages, Updated Start New Message Flow]

### ISSUE THREAD

**Issue by steveg-IAT** [department-of-veterans-affairs/va.gov-team#117951]
### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

- @steveg-IAT 

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")
- [ ] (Mobile app only) Color contrast ratios are met for all supported dark, contrast, and inverted color modes
- [ ] (Mobile app only) All elements retain their meaning for all supported dark, contrast, and inverted color modes

### How did color testing go?

- (Globally) Focus Indicator: Yellow focus indicator does not meet contrast standards

### Who completed the axe scans?

- @steveg-IAT ran axe scans manually on each of the three flow URLs

### axe checks

- [x] Each page or screen has been scanned using axe (results shared below)
- [x] axe is integrated into your end-to-end testing

### axe DevTools scan results

- See [comment below for screenshot of results](https://github.com/department-of-veterans-affairs/va.gov-team/issues/117951#issuecomment-3229898172) 

### How did axe testing go?

- Passed all axe testing

### Who completed the content zoom and reflow test?

- @steveg-IAT 

### Content zoom and reflow checks

- [x] (Web only) All page elements are readable and usable at 200% zoom
- [x] (Web only) All page elements are readable and usable at 300% zoom
- [x] (Web only) All page elements are readable and usable at 400% zoom
- [ ] (Mobile app only) All page elements are readable and usable when device text is at the maximum possible size

### How did content zoom and reflow testing go?

- Accordion Trigger: "Only use messages for non-urgent needs": @300% open/close indicator orphaned
- Accordion Trigger: "What to know about attaching files": @400% open/close indicator orphaned
- Lost Visible Focus: 1280x1310@400% Back-to-top click correctly takes you page-top, highlights "Start message" H1, but is not or barely visible on the page (below fold)

### Who completed the keyboard navigation test?

- @steveg-IAT 

### Keyboard navigation checks

- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [x] All elements under focus have a visible focus indicator
- [x] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

- Button: Select Care Team drop-down's button caret does receive focus and is not usable via keyboard controls, though the drop-down works as expected otherwise, inline with system select field.

### Do you have any other results to share?

- See [comment below for additional items noticed during testing](https://github.com/department-of-veterans-affairs/va.gov-team/issues/117951#issuecomment-3229866047)

**Comment by sterkenburgsara** [comment]
Feature label not applicable for us on this work

**Comment by steveg-IAT** [comment]
Did not test the mobile app, web/responsive only.
Test User: [[cut]]+gustav @ gmail.com (lost access via this acct at the end of the day - 20250827)

- **http://staging.va.gov/my-health/secure-messages/new-message**
  - Link: 911: target size 25x22 (2.5.8 Target Size (Minimum) (Level AA) 24x24)
  - Breadcrumb: "Back"
- **https://staging.va.gov/my-health/secure-messages/new-message/select-care-team**
  - Accordion Trigger: "Only use messages for non-urgent needs": @300% open/close indicator NOT orphaned
  - Breadcrumb: "VA.gov home > My HealtheVet"
- **https://staging.va.gov/my-health/secure-messages/new-message/start-message**
  - Page Content Spacing inconsistent (margin larger than select-care-team $small-screen and under)
  - Breadcrumb: "Back"
  - Accordion Trigger: "Only use messages for non-urgent needs": @300% open/close indicator orphaned
  - Accordion Trigger: "What to know about attaching files": @400% open/close indicator orphaned
  - Button: Attach File: inconsistent button style; no visible hover state (in/near group of primary CTA buttons (?))
  - White/Blue Buttons have very little noticeable contrast change on hover when using monochrome settings
  - Button: Connect with the Veterans Crisis Line: inconsistent hover state (or different non-white background(?))
  - Lost Visible Focus: 1280x1310@400% Back-to-top click correctly takes you page-top, highlights "Start message" H1, but is not or barely visible on the page (below fold)
  - BuBack-to-top 


## Notes/Questions
- Link: 911: Should this be a button like the "crisis line"
- Select Care Team drop-down (select + Input) different component than Category drop-down (select)?
  - Keyboard Navigation: these drop-downs differ in keyboard functionality
  - The open menu options differ in design, as well
- Should the user be able to tab to the Select Care Team drop-down's caret, it's a clickable button??

**Comment by steveg-IAT** [comment]
AXE-Report screencaps:

- <img width="1254" height="262" alt="Image" src="https://github.com/user-attachments/assets/60b9b135-0509-49d6-aa28-9f24396ca03f" />
- <img width="1251" height="263" alt="Image" src="https://github.com/user-attachments/assets/f1daf5b0-5655-42ed-a8f7-ef16ee6719a2" />
- <img width="1249" height="262" alt="Image" src="https://github.com/user-attachments/assets/f36a2a9a-0071-4b2f-b395-29b59bba8c0c" />
